base/src/lib.rs

```diff
@@ -242,6 +242,23 @@ pub enum TpmiAlgRsaScheme{
     OAEP = TpmAlgId::OAEP.0,
 }
 
+/// TpmiAlgSigScheme represents values that may appear in the scheme parameter of a TpmtSigScheme (TPMI_ALG_SIG_SCHEME).
+/// See definition in Part 2: Structures, section 9.37.
+#[open_enum]
+#[repr(u16)]
+#[rustfmt::skip] #[derive(Debug)] // Keep debug derivation separate for open_enum override.
+#[derive(Copy, Clone, PartialEq, Default, Marshalable)]
+#[allow(clippy::upper_case_acronyms)]
+pub enum TpmiAlgSigScheme {
+    RSAPSS = TpmAlgId::RSAPSS.0,
```

Comment:
Our formatting of these things is inconsistent... Not your problem, just apologizing for the mess. Thanks for the PR!